api/app/wildfire_one.py

```diff
@@ -129,14 +123,41 @@ async def _fetch_raw_stations(session: ClientSession, headers: dict, query_build
         page_count = page_count + 1
 
 
+async def _fetch_detailed_geojson_stations(
+        session: ClientSession,
+        headers: dict,
+        query_builder: BuildQuery) -> (Dict[int, GeoJsonDetailedWeatherStation], Dict[str, int]):
+    stations = {}
+    id_to_code_map = {}
+    # Put the stations in a nice dictionary.
+    async for raw_station in _fetch_raw_stations_generator(session, headers, query_builder):
+        station_code = raw_station.get('stationCode')
+        station_status = raw_station.get('stationStatus', {}).get('id')
+        # Because we can't filter on status in the RSQL, we have to manually exclude stations that are
+        # not active.
+        if _is_station_valid(raw_station):
```

Comment:
I'm not understanding how this `if` statement is filtering out the inactive stations?